lib/pbench/server/api/resources/graphql_schema.py

```diff
@@ -0,0 +1,74 @@
+from pbench.server.api.resources.models import MetadataModel
+from pbench.server.api.resources.database import Database
+
+import graphene
+from graphene import relay
+from graphene_sqlalchemy import SQLAlchemyObjectType
+
+
+# Define graphql types
+class Metadata(SQLAlchemyObjectType):
+    class Meta:
+        model = MetadataModel
+        interfaces = (relay.Node,)
+
+
+class MetadataAttribute:
```

Comment:
We should consider making the `config` and `description` fields required using the Graphene APIs in order to verify transactions involving this type.